paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/CustomerSettingsDefinition.kt

```diff
@@ -8,6 +8,8 @@ internal object CustomerSettingsDefinition :
     PlaygroundSettingDefinition<CustomerType>,
     PlaygroundSettingDefinition.Saveable<CustomerType>,
     PlaygroundSettingDefinition.Displayable<CustomerType> {
+    private var previouslyUsedId: String? = null
```

Comment:
We shouldn't store this here. `CustomerSettingsDefinition` is an object, making this a global.

We can update the snapshots value to include the customer for this key instead: https://github.com/stripe/stripe-android/blob/d80b6413f08a04d2ff9ec895aa5e6ae97b7fe045/paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/CustomerSettingsDefinition.kt#L44

Happy to pair on this too if you want.

Reply:
Updated here: https://github.com/stripe/stripe-android/pull/8352